login/README.md

```diff
@@ -68,3 +68,39 @@ glome-login uses error tags to communicate errors.
 This error means that `glome-login` could not figure out what service key to
 use. This most likely means that you have not specified a service key in the
 configuration file (by default `/etc/glome/config`).
+
+# PAM module
+
+`pam_glome.so` library implements the PAM authentication module for the
+[GLOME Login](../docs/glome-login.md) protocol.
+
+## Installation
+
+1. Install the library into the system dependent location for PAM modules
+   (for example `/lib/security/pam_glome.so`).
+1. Enable and configure PAM module for a specific service (for example
+   `/etc/pam.d/login`):
+
+```
+auth       requisite  pam_glome.so
+```
+
+## Usage
+
+PAM module supports the following options:
+
+* `config_path=PATH` - location of the configuration file to parse (defaults to
+  `/etc/glome/config`)
+* `service_key=KEY` - use hex-encoded `KEY` as the service key (defaults to key
```

Comment:
Don't we also need a key ID in this case?

Reply:
Added a new option `service_key_version` for setting the key ID.